README.md

```diff
@@ -69,7 +69,7 @@ Run `./install_deps.sh` to install the JavaScript dependencies and the Python de
 #### First Time Setup
 
 1.  Go to backend directory: `cd backend`
-1.  Run `make install` to install all backend libraries.
+1.  Run `pdm install` to install all backend libraries.
```

Comment:
Sure, I can change that. 

Since pdm uses the PEP standard it would actually not really be required to have pdm just for running the backend since you could do `pip install .`. that would require some code refactoring though to root all code in one package, probably not worth it for now.

Reply:
Yep, sounds like telling people to install pdm ist the simpler solution for now.